### PR TITLE
Release update BE-1897

### PR DESCRIPTION
**Pull request by CanKarabulut**

### CODE REVIEW

File: docs/build/build-profile-configuration.md [hunk 1]
```diff
@@ -27,6 +27,28 @@ You may create **Distribution** configuration to send your app to public stores
...
+Then click on the three dots next to the configuration we want to copy, and click on the "Clone" button in the mini window that opens.
+
+![](<https://cdn.appcircle.io/docs/assets/clone-2.png>)
+
+And another one is created identical to the configuration we want to clone. 
+
+![](<https://cdn.appcircle.io/docs/assets/clone-3.png>)
+
+:::info
+The naming here is created by adding "_copy_1" to the end of the main configuration name. For each configuration to be copied from now on, the naming will be incremented as "_copy_ 1-2-3" in the form of "_copy_ 1-2-3... " incrementally.
```
Comment (csonuryilmaz) [inline docs/build/build-profile-configuration.md:49]: We can mention that although the system gives a default unique name for the cloned configuration the user can **rename** it to any name he/she wants.

**Review by csonuryilmaz — CHANGES_REQUESTED** [review]

File: docs/build/adding-a-build-profile/connecting-multiple-instance.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title: 'Connecting to Multiple Instance'
+metaTitle: 'Multiple Instances Using Git Providers (Bitbucket, Gitlab, Azure)'
```
[the remark this answers is not in the document]
Comment (CanKarabulut) [reply docs/build/adding-a-build-profile/connecting-multiple-instance.md:3]: Fixed

File: docs/build/adding-a-build-profile/connecting-multiple-instance.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+metaDescription: 'Multiple Instances Using Git Providers (Bitbucket, Gitlab, Azure)'
```
[the remark this answers is not in the document]
Comment (CanKarabulut) [reply docs/build/adding-a-build-profile/connecting-multiple-instance.md:4]: Fixed

File: docs/build/adding-a-build-profile/connecting-multiple-instance.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+import ContentRef from '@site/src/components/ContentRef';
+
+### Multiple Instances Using Git Providers (Bitbucket, Gitlab, Azure)
+
+Multiple connection features have been introduced for connections made with PAT (Personnel Access Token) on Bitbucket, Azure, or Gitlab. Thus, it is possible to add Bitbucket, Azure, or Gitlab servers located in different environments by a single user.
+
+For example, the same user can connect to two different instances, such as dev1.azure.campaignname.com and
+We can also give it to dev2.azure.campaignname.com.
+
+Similar examples are dev1.gitlab.campaignname.com and dev2.gitlab.campaignname.com or dev.bitbucket.campaignname.com and prod.bitbucket.campaignname.com.
```
[the remark this answers is not in the document]
Comment (CanKarabulut) [reply docs/build/adding-a-build-profile/connecting-multiple-instance.md:16]: Fixed

File: docs/build/build-profile-configuration.md (hunk 1 above)
Comment (CanKarabulut) [reply docs/build/build-profile-configuration.md:49]: fixed

File: docs/build/adding-a-build-profile/connecting-multiple-instance.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+![](<https://cdn.appcircle.io/docs/assets/azure-m-repo.png>)
+
+Then click on the "Connect to an Azure DevOps Server" button in the window that opens;
+
+![](<https://cdn.appcircle.io/docs/assets/azure-m-repo-1.png>)
+
+In the next window, fill in the relevant fields and click on the "Connect" button.
+
+![](<https://cdn.appcircle.io/docs/assets/azure-m-repo-2.png>)
```
[the remark this answers is not in the document]
Comment (CanKarabulut) [reply docs/build/adding-a-build-profile/connecting-multiple-instance.md:34]: fixed

File: docs/build/adding-a-build-profile/connecting-multiple-instance.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+import NeedHelp from '@site/docs/\_need-help.mdx';
+
+<NeedHelp />
```
[the remark this answers is not in the document]
Comment (CanKarabulut) [reply docs/build/adding-a-build-profile/connecting-multiple-instance.md:38]: fixed

File: docs/workflows/why-to-use-workflows.md [hunk 7]
```diff
@@ -137,9 +137,34 @@ Each workflow step has its own set of configuration options, which can be set by
...
-- **Continue with the next step even if this step fails:** If a step is optional or if its result should not cause a build failure, you can select this option to continue the workflow in case of the failure of this specific step. In the default workflows, this option is turned on for certain steps.
-- **Workflow Step Version: **You can select a specific version of a step with which to execute your build. If you select a version with an asterisk (\*), you will receive the minor updates to the workflow step automatically. The major versions may include added or removed input fields and manual version selection is required for major version updates.;
+
+- **Continue with the next step even if this step fails:** If a step is optional or its result should not cause a build error, you can select this option to continue the workflow if this particular step fails. In default workflows, this option is on for specific steps. And since this step is active, the build status will appear as "Warning" when other steps in the build are successful. (Detailed explanation about this step is given at the bottom).
```
[the remark this answers is not in the document]
Comment (CanKarabulut) [reply docs/workflows/why-to-use-workflows.md:141]: fixed

File: docs/build/adding-a-build-profile/connecting-multiple-instance.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+title: 'Connecting to Multiple Instance'
+metaTitle: 'Multiple Instances Using Git Providers (Bitbucket, Gitlab, Azure)'
+metaDescription: 'Multiple Instances Using Git Providers (Bitbucket, Gitlab, Azure)'
+---
+
+import ContentRef from '@site/src/components/ContentRef';
+
+### Multiple Instances Using Git Providers (Bitbucket, Gitlab, Azure)
+
+Multiple connection features have been introduced for connections made with PAT (Personnel Access Token) on Bitbucket, Azure, or Gitlab. Thus, it is possible to add Bitbucket, Azure, or Gitlab servers located in different environments by a single user.
```
[the remark this answers is not in the document]
Comment (CanKarabulut) [reply docs/build/adding-a-build-profile/connecting-multiple-instance.md:11]: fixed

[the remark this answers is not in the document]
Comment (CanKarabulut) [reply docs/build/adding-a-build-profile/connecting-multiple-instance.md:11]: To add a git provider, PAT support is sufficient. There is no distinction between self-hosted and cloud-based.

File: docs/workflows/ios-specific-workflow-steps.md [hunk 9]
```diff
@@ -17,6 +17,23 @@ This step is used to specify the Xcode version to be used during the build proce
 
 https://github.com/appcircleio/appcircle-xcode-select-component
 
+### Pool-Based Xcode Version Selection
```
[the remark this answers is not in the document]
Comment (CanKarabulut) [reply docs/workflows/ios-specific-workflow-steps.md:20]: fixed

**Review by csonuryilmaz — APPROVED** [review]
LGTM! :pray:

**Pull request merged**